src/constants.rs

```diff
@@ -3,4 +3,6 @@ use std::num::NonZeroUsize;
 // SAFETY: Safe because value non zero
 pub(crate) const MAX_BITS: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(255) };
 // SAFETY: Safe because value non zero
+pub(crate) const NUM_HASH_BITS: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(250) };
```

Comment:
Why 250? For [u8; 32] it will be 256 bits

Reply:
This is the number defined in Nova, any large enough number of bits would be fine

Reply:
Well I'd go for 256, otherwise it looks like a weird disregard for the last 6 bits